Assets/MixedRealityToolkit/_Core/Utilities/UX/HoverLight.cs

```diff
@@ -92,21 +100,87 @@ private void OnDrawGizmosSelected()
             Gizmos.DrawIcon(transform.position + Vector3.back * Radius, string.Empty, false);
         }
 
-        private void Initialize()
+        private static void AddHoverLight(HoverLight light)
```

Comment:
Is there a reason why we are introducing so many static functions?
If there is need for a single instance of the script, then it should probably be a runtime component.

Can we clarify why these need to be static @Cameron-Micka ?

Reply:
None of these "need" to be static. I just like to mark them static to hint to other programmers that these functions will likely only operate on what is passed in, or other static members. It's also probably a premature optimization haha.

Reply:
Ok, can we change these back to non-static and we can review as required @keveleigh

Reply:
O think it makes sense that they're statics seeing as we only have three lights to work with at any time.

Reply:
Already removed the statics in the PR @StephenHodgson, also note these are private statics.
Lets review once this starts getting used, unless you feel the statics should remain?